content/src/view.js

```diff
@@ -2626,17 +2681,34 @@ function setPaneEditorBlockOptions(pane, pal, modeOptions) {
   }
 }
 
-function setPaneEditorBlockMode(pane, useblocks) {
+function setPaneEditorBlockMode(pane, useblocks, editor) {
+  function setMainEditorBlockMode(editor, useblocks) {
+    if (editor.currentlyUsingBlocks == useblocks) return false;
```

Comment:
I tend to avoid skipping curly braces even for single-line conditionals, `{ return false; }`.  Thanks!